test_task_journey.py

```diff
@@ -0,0 +1,8 @@
+import os
+import sys
+from firefox_driver import driver
+import test_app
+
+def test_task_journey(driver, test_app):
+    driver.get('http://localhost:5000/')
+    assert driver.title == 'To-Do App'
```

Comment:
Can you try including some page interaction?

Reply:
Sure, I will do that now. I was unable to do that before because I wasn't able to get the web driver working correctly.

Reply:
This is still not working. It is now able to create the driver, but invoking `driver.get` causes an exception;
```
response = {'status': 500, 'value': '{"value":{"error":"unknown error","message":"Reached error page: about:neterror?e=connection...ate@chrome://marionette/content/listener.js:297:13\\nhandleEvent@chrome://marionette/content/listener.js:263:14\\n"}}'}
```

Reply:
I finally got it to work. It seems that it is a problem with WSL2. What I needed to do was to create an Ubuntu Server machine in Azure, Install a visual desktop, enable RDP, RDP into the machine, install the development environment, install Firefox, then run the test...
![Screenshot_2020-09-17_20-57-21](https://user-images.githubusercontent.com/13692795/93527625-0e7ce000-f931-11ea-9dd6-35863b0b60df.png)

Reply:
OK I took it one step further. I installed the Remote Development extension and connected to my new Ubuntu machine from VS Code with SSH. Then the test worked as well!
![image](https://user-images.githubusercontent.com/13692795/93529314-9bc13400-f933-11ea-8e9a-00f15c700986.png)

Reply:
I'm going to try one last mad thing. I'm going to try doing remote development to my raspberry pi as the host, which means I can do away with my cloud VM.